### PR TITLE
Fai to bed using awk

### PR DESCRIPTION
**Pull request by asp8200**
Issue #5132

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/custom/faitobed/meta.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
```
Comment (Krannich479) [inline modules/nf-core/custom/faitobed/meta.yml:2]: L:1-2 aren't necessary

File: modules/nf-core/custom/faitobed/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
```
Comment (Krannich479) [inline modules/nf-core/custom/faitobed/tests/main.nf.test:1]: same here with all the TODOs

**Comment by maxulysse** [comment]
Closing in favor of using an already existing container

**Comment by maxulysse** [comment]
cf https://github.com/nf-core/sarek/pull/1439

**Pull request closed without merging**